ansible_runner/utils.py

```diff
@@ -240,12 +240,14 @@ def write(self, data):
                 if not self.suppress_ansible_output:
                     sys.stdout.write(stdout_actual + "\n")
```

Comment:
This is what is still not being flushed. Adding a line after `sys.stdout.flush()` made things work as expected.